### PR TITLE
Add docs about how to add custom data attributes

### PR DESCRIPTION
**Pull request by gforcada**
Changes proposed in this pull request:

Today I discovered that one can add its own custom ``data-`` attributes and decided to document it, for everyone's benefit 😄

### CODE REVIEW

**Comment by gforcada** [comment]
@plone/documentation-team if there is a better place where to put this docs, please let me know!

File: develop/addons/javascript/data-attributes.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+.. note::
+
+   If done like this, all options will be prefixed ``data-pat-``.
+   This is the standard way to configure/tweak mockup/patternslib patterns (hence the ``pat-`` prefix)
+
+Adapter
+^^^^^^^
+If the ``data-pat-`` prefix does not suite you, or you want complete freedom on what to add,
+you can create a ZCA adatapter.
```
Comment (stevepiercy) [inline develop/addons/javascript/data-attributes.rst]: adapter

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Please fix misspelled word

File: develop/addons/javascript/data-attributes.rst (hunk 1 above)
Comment (gforcada) [reply develop/addons/javascript/data-attributes.rst]: 😊 oops!

**Review by stevepiercy — APPROVED** [review]

**Comment by svx** [comment]
Adjusted and merged !

Thanks !

**Comment by gforcada** [comment]
@svx thanks for reviewing and merging!!

**Pull request merged**